test/test_jit_fuser_te.py

```diff
@@ -1229,12 +1229,14 @@ def apply(fn):
             torch.bool,
         ]
         unary_ops = [
```

Comment:
Not for this PR:  it would be nice to validate  that we have actual run all overloads in our tests. 

We could try to make  the `isSupported()` set queryable, and then validate within `test_jit_fuser_te.py` that all all overloads have been fused by making the `assertLastGraphAllFused` add the schemas of fused nodes to some list.

Reply:
Yeah, agreed. I couldn't come up with a good way to do that yet.